package.json

```diff
@@ -26,38 +26,38 @@
     "dist"
   ],
   "dependencies": {
-    "popper.js": "^1.14.4"
+    "popper.js": "^1.15.0"
   },
   "devDependencies": {
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "^10.0.2",
     "chai": "^4.2.0",
     "chai-enzyme": "1.0.0-beta.1",
-    "condition-circle": "1.5.0",
+    "condition-circle": "2.0.2",
     "dont-crack": "1.2.1",
-    "enzyme": "3.7.0",
-    "enzyme-adapter-react-16": "1.6.0",
-    "eslint": "^5.6.1",
-    "eslint-plugin-cypress-dev": "^1.1.2",
-    "eslint-plugin-mocha": "^5.2.0",
-    "eslint-plugin-react": "^7.11.1",
-    "fs-extra": "7.0.0",
+    "enzyme": "3.10.0",
+    "enzyme-adapter-react-16": "1.14.0",
+    "eslint": "^5.16.0",
+    "eslint-plugin-cypress-dev": "^2.1.0",
+    "eslint-plugin-mocha": "^5.3.0",
+    "eslint-plugin-react": "^7.13.0",
+    "fs-extra": "8.0.1",
     "github-post-release": "1.13.1",
     "jsdom": "^9.5.0",
     "lodash": "^4.17.11",
-    "prop-types": "^15.6.2",
+    "prop-types": "^15.7.2",
     "react": "16.5.2",
     "react-dom": "16.5.2",
-    "semantic-release": "8.2.0",
-    "simple-commit-message": "3.3.1",
-    "sinon": "^6.3.5",
-    "sinon-chai": "^3.2.0",
-    "zunder": "6.2.0"
+    "semantic-release": "15.13.16",
```

Comment:
Upgrading `semantic-release` could be a problem. I think 9.0 changed it somewhat dramatically and it stopped working with `simple-commit-message` and `github-post-release` without updating the configuration. Of course, it's hard to tell until we actually try to release a new version.

Reply:
[@cypress/browserify-preprocessor](https://github.com/cypress-io/cypress-browserify-preprocessor/blob/master/package.json) is an example of upgrading `semantic-release`. See the `release` configuration at the bottom. Though it never behaves the way I expect it. I think using `feat: some feature` doesn't work anymore.

Reply:
Oh yeah, good catch. I just updated all the dependencies because something, somewhere was requiring the vulnerable `event-stream` version and breaking, so moved this back to `8.2.0`.